coremltools/converters/nnssa/coreml/ssa_converter.py

```diff
@@ -2232,6 +2232,143 @@ def _convert_layer_normalization(self, node):
         self.tensor_shapes[node.name] = self._get_tensor_shape_from_type(
             node.datatype)
 
+    def _convert_binary(self, node):
+        """
+        Convert binary operator
+            - Attempts to add elementwise operator if possible
+            - Otherwise, inserts broadcastable operator
+        """
+        def _is_elementwise_scalar_check(input_type):
```

Comment:
I wouldn't worry about prepending `_` to inner methods names. I'm almost certain inner methods can only be accessed from the outer method.